open-budget/app/styles/main.less

```diff
@@ -4,28 +4,39 @@
 //@import url("http://fonts.googleapis.com/earlyaccess/alefhebrew.css");
```

Comment:
@akariv fixed in 59dc1a1d571de55acc85cc94d32173c995ce615e
This was a bit challenging - it probably calls for main.less to be broken up to smaller files.